widgetastic_manageiq/__init__.py

```diff
@@ -4705,3 +4705,19 @@ def read_additional_info_popup(self, vm_name):
             'Started': self.browser.text('./div[2]/div/div[1]', parent=el),
             'Description': self.browser.text('./div[2]/div/div[2]', parent=el),
             'Conversion Host': self.browser.text('./div[2]/div/div[3]', parent=el)}
+
+
+class HiddenFileInput(BaseFileInput):
+    """Uploads file via hidden input form field
+
+    Prerequisite:
+        Type of input field should be file (type='file')
+    """
+    def fill(self, filepath):
+        self.browser.set_attribute("style", "display;", self)
```

Comment:
**REQUIRED**
Is that semicolon really needed in `display;`?

Reply:
@quarckster  removing `;` will also not make much changes but here I am inserting css element in html page so thought to close that element there itself so it should not combine with other css element while rendering, just for precaution.

Reply:
I don't think it's necessary. You can remove it.